src/mount.ts

```diff
@@ -13,6 +13,6 @@ export default (component: RootTramOneComponent, container: Container) => {
 
 	// this sadly needs to be wrapped in some element so we can process effects
 	// otherwise the root node will not have effects applied on it
-	const renderedApp = html`<div><app /></div>`;
+	const renderedApp = html`<tram-one><app /></tram-one>`;
```

Comment:
In the UI, this appears like the following:
![image](https://user-images.githubusercontent.com/326557/175829118-a339f5c5-4afe-4448-a336-a40f7800367c.png)

This is better than `<div>`, because we won't inherit any unexpected styles or formatting.